### ISSUE TITLE
Image on duckerhub needs updated with latest SRB2Kart

### ISSUE THREAD

**Issue by mmarsalko** [BrianAllred/srb2kart#1]
First off, thanks so much for making this image. I've been making use of it for months on my 'kart server. I actually was in the process of making my own, but it had some bugs, and yours "just worked" :)

A new version of SRB2Kart released this week (v1.2), and the image on dockerhub is now out of date, so it needs a push. In the future, maybe we can collaborate on building some CI in GitLab so new images are automatically built and pushed when a new version of kart pops up on the apt repo.

For the meantime, I've gone ahead and built the image myself locally, so there's no rush to get this done.

**Comment by BrianAllred** [comment]
Yeah, migrating to Gitlab and setting up automated CI would be nice. I'll leave this issue open to remind me to do it someday. For now, I've pinned the Dockerfile to Ubuntu 18.04 (since it was failing to build with latest) and pushed a new build.

**Comment by Rax14207** [comment]
Version 1.3 of SRB2Kart released this week.  Any chance the a new build could be pushed?  I attempted to build it locally, but ran into some issues (Still learning how to use Docker and such...).

**Comment by HiddenKitten** [comment]
1.3 over on #2, hopefully, it's only a temporary fix that eventually won't be necessary.

@Rax14207 you should be able to build it with my source, they didn't update a couple files on the PPA, so they need to be acquired manually, at any rate, hopefully KartKrew fixes the PPA sooner, rather than later, but this should work in the meantime: https://github.com/HiddenKitten/srb2kart

this will fix if your issue was something like "patch.kart was corrupted or modified..." when trying to run the image.

**Comment by Rax14207** [comment]
@HiddenKitten Thanks!  "patch.kart was corrupted or modified..." was definitely the last issue I had (after a bit of issues with just trying to build locally for the first time).  I was able to build using your source and everything is good now.

**Comment by BrianAllred** [comment]
Merged PR#2.
